src/sync_dashboards/main.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+import yaml
+
+import click
+import looker_sdk
+
+from looker_sdk import methods, models
+
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def setup_sdk(client_id, client_secret, instance) -> methods.Looker31SDK:
+    os.environ["LOOKERSDK_BASE_URL"] = instance
+    os.environ["LOOKERSDK_API_VERSION"] = "3.1"
+    os.environ["LOOKERSDK_VERIFY_SSL"] = "true"
+    os.environ["LOOKERSDK_TIMEOUT"] = "120"
+    os.environ["LOOKERSDK_CLIENT_ID"] = client_id
+    os.environ["LOOKERSDK_CLIENT_SECRET"] = client_secret
+
+    return looker_sdk.init31()
+
+
+def sync_dashboards(sdk: methods.Looker31SDK, config: dict) -> None:
+    for dashboard_id, lookml_dashboard_id in config.items():
+        try:
+            dashboard = sdk.dashboard(str(dashboard_id))
+            sdk.dashboard_lookml(lookml_dashboard_id)
+            if dashboard.lookml_link_id is None:
+                sdk.update_dashboard(
+                    str(dashboard_id),
+                    models.WriteDashboard(lookml_link_id=lookml_dashboard_id),
+                )
+            elif dashboard.lookml_link_id != lookml_dashboard_id:
+                logging.warning(
+                    f"Dashboard {dashboard_id} lookml_link_id changing from {dashboard.lookml_link_id} to {lookml_dashboard_id}"
+                )
+                sdk.update_dashboard(
+                    str(dashboard_id),
+                    models.WriteDashboard(lookml_link_id=lookml_dashboard_id),
+                )
```

Comment:
Are there any downsides to updating the dashboards every time?
```suggestion
            sdk.update_dashboard(
                str(dashboard_id),
                models.WriteDashboard(lookml_link_id=lookml_dashboard_id),
            )
```

Reply:
This should be fine to do.